index.html

```diff
@@ -1,14 +1,23 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="utf-8">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/5.0.0/normalize.css">
+    <link rel="stylesheet" href="https://unsplash.com/search/photos/bicycle">
```

Comment:
Is this needed? I think we should either link directly to the site with absolute paths or download them locally for relative paths.